### PR TITLE
Improve the workspace

### PR DESCRIPTION
**Pull request by rianby64**
- Remove an unnecessary dependency: "devtools-terminal".
- Add the support for .es extensions to VSCode

## Motivation and Context
devtools-terminal is not compatible with linux environments. If run `npm install` then a bunch of errors will appear and this fact obstructs the development in other platforms different to MacOS. 

## How Has This Been Tested?
After removing that dependency, I ran all the scripts and all of them didn't fail. Looks like that dependecy can be safely dropped.

### CODE REVIEW

File: configure/README.md [hunk 1]
```diff
@@ -5,3 +5,30 @@ Scripts used for configuring server nodes
 ## `nginx`
 
 nginx setup. Derived from [`whatwg/misc-server`](https://github.com/whatwg/misc-server)
+
+## `vscode`
```
Comment (brandondees) [inline configure/README.md]: is this section necessary to add in addition to the config file?

if so, does it really belong in the main README or could it go into a separate doc page?

Comment (snuggs) [reply configure/README.md]: Hmmm @brandondees @rianby64 you both are right. It IS part of "configuration" per say. The thought was anything "configure" related stuff goes in `/configure`. But that is more internal dev related (perhaps shouldn't be?). IMHO this documentation should be as close to introduction as possible. Perhaps an IDE section? /cc @tmornini 

In addendum i'll add what I did for VIM.

Thanks for this @rianby64

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
   },
   "scripts": {
     "test": "bin/test",
-    "start": "bin/serve",
+    "start": "bin/browse",
```
Comment (snuggs) [inline package.json]: I think we should leave this as serve as Heroku would blowup with browser sync.

File: package.json [hunk 3]
```diff
@@ -57,7 +57,6 @@
     "postcss-cli": "*",
     "autoprefixer": "*",
     "browser-sync": "*",
-    "devtools-terminal": "*",
```
Comment (snuggs) [inline package.json]: @rianby64 Also have updated some dependencies due to security risk. Check for merging of #181 which may cause (easy) conflict that may need fixing. But DEFINITELY want to get rid of this.

File: configure/README.md [hunk 4]
```diff
@@ -5,3 +5,30 @@ Scripts used for configuring server nodes
...
+2. Open the "Workspace settings" tab
+3. Add the property "files.assiciations" if not present or extend it with the value "*.es": "javascript"
+
+Should looks like
+
+```javascript
+    "files.associations": {
+        "*.es": "javascript"
+    }
+```
```
Comment (snuggs) [inline configure/README.md]: EASY PEASY @rianby64. This is great. Now where it should go see above. But wherever it goes this is great.

File: package.json [hunk 5]
```diff
@@ -34,10 +34,10 @@
     "example": "examples"
   },
   "scripts": {
-    "test": "bin/test",
-    "start": "bin/browse",
-    "package": "bin/package",
-    "postinstall": "bin/publish"
+    "test": "bin/snuggsi test",
```
Comment (snuggs) [inline package.json]: @brandondees @rianby64 @tmornini wow can resolve merge conflicts from Github now. :-)

@rianby64 took care of it and merged into your branch as it was my doing from merging #175 earlier today.

**Comment by snuggs** [comment]
@rianby64 yeah man we gotta get rid of devtools.
capture d ecran 2018-12-10 a 21 59 28

File: package.json (hunk 3 above)
Comment (rianby64) [reply package.json]: Bad news. If you rely on "devtools-terminal" then please, check the following:
https://github.com/chjj/pty.js#todo
> Add a way of determining the current foreground job for platforms other than Linux and OSX/Darwin.

`devtools-terminal` relies on `pty` which works only on Mac.

Comment (snuggs) [reply package.json]: @rianby64 EXACTLY. Let's kill it with fire. To be honest it only was around when I was testing VIM in browser.  I'm going to go ahead and merge this.

**Comment by snuggs** [comment]
@rianby64 please see #183

**Pull request closed without merging**